internal/commands/app/create_inputs.go

```diff
@@ -322,7 +329,11 @@ func (i createInputs) args(omitDryRun bool) []flags.Arg {
 		args = append(args, flags.Arg{flagLocalPathCreate, i.LocalPath})
 	}
 	if i.Template != "" {
-		args = append(args, flags.Arg{flagTemplate, i.Template})
+		if i.Template == noArgsDefaultValueTemplate {
+			args = append(args, flags.Arg{Name: flagTemplate})
```

Comment:
remove `<prompt-template>` when we display the command if a user declared --template with no args